website/src/content/api-reference/deployment-previews.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you test your Custom Application features before publishing them to the production environment.
+
+</Subtitle>
+
+This document describes the necessary steps to build and add deployment previews to your Custom Application in the Merchant Center.
+
+The process involves:
+
+1. Building deployment previews for your Custom Application.
+2. Deploying the Custom Application preview to your hosting provider.
+3. Adding the deployment preview to the Custom Application settings.
+
+# Building deployment previews
+
+Each deployment preview has its unique and stable deployment URL. To build a deployment preview, you must set the `env.production.url` value in the `custom-application-config.json` file to the deployed application. You can follow one of the below approaches:
```

Comment:
> You can follow one of the below approaches:

Aren't both steps required? Maybe this section should be reworked.

Reply:
Nope. But you can use them in conjunction. 

The ENV approach isn't dependent on Git, you can have different env variables on your hosting provider, and it'll work perfectly. 

Same goes for the Git approach, if you hardcode the values, you need to keep them separate with branching. In this case, the hosting provider relies on the value present in the build branch (configured on the hosting platform side)

Reply:
@emmenko I’ve removed this section to avoid confusion, as we discussed.